### ISSUE TITLE
Smiley keyboard overlaps

### ISSUE THREAD

**Issue by karthikAdaptavant** [zummenix/KeyboardWrapper#14]
- when i tap smile keyboard its hiding the toolbar
- when i change the keyboard settings to predictive, toolbar overlapping with predictive suggestions 
<img width="377" alt="screen shot 2017-10-19 at 6 53 40 pm" src="https://user-images.githubusercontent.com/11072850/31773053-2d25ba50-b4ff-11e7-8445-60fc5cda5eb2.png">
<img width="392" alt="screen shot 2017-10-19 at 6 57 26 pm" src="https://user-images.githubusercontent.com/11072850/31773147-6cfaf85c-b4ff-11e7-9401-a6f9be3a784e.png">
<img width="405" alt="screen shot 2017-10-19 at 6 57 41 pm" src="https://user-images.githubusercontent.com/11072850/31773151-6d5c11dc-b4ff-11e7-817c-250c04f4cb46.png">

**Comment by zummenix** [comment]
I wasn't able to reproduce this. Do you use any specific code there?

**Comment by karthiikmk** [comment]
No, i did not add any lines of code. I long pressed smiley tab (in keyboard) -> selected keyboard settings (it took me to the settings page) -> turned on the predictive keyboard, then I came back to the app. the above issue happened

**Comment by zummenix** [comment]
Ah, I see the problem. Short investigation showed that appropriate notifications aren't fired by iOS. I've found similar stackoverflow question https://stackoverflow.com/questions/46287136/uikeyboardwillchangeframe-notification-not-called-with-emoji-keyboard

I'm not sure how to fix it properly since in my opinion it's a bug in iOS.

@karthikAdaptavant Thank you for reporting the issue, I wouldn't notice it. I hope someone will find a solution. 

I'll leave the issue open for now.
